junit/src/main/java/org/kaazing/k3po/junit/rules/Latch.java

```diff
@@ -129,19 +132,19 @@ boolean hasException() {
 
```

Comment:
NotifyException and setInterrrupteOnException are exactly the same? if so they should be combined